nixos/modules/services/networking/postgrest.nix

```diff
@@ -0,0 +1,114 @@
+{
+  lib,
+  pkgs,
+  config,
+  ...
+}: let
+  cfg = config.services.postgrest;
+  inherit (lib) mkOption types literalExample mdDoc;
+in {
+  options.services.postgrest = {
+    enable = lib.mkEnableOption (mdDoc "postgrest");
+
+    dbURI = mkOption {
+      type = types.nullOr types.str;
+      default = null;
+      example = "postgres://username:password@localhost:5432/books";
```

Comment:
NixOS heavily prefers socket paths so the example should contain that, too if possible.

Reply:
or is this not possible?

Reply:
It is possible, the socket dir path must be `%2F` separated instead of `/`. I recently did this in https://github.com/juspay/todo-app/blob/76b5a56c7811b79ff6556be1c8cd79b62f12432b/nix/services/postgrest.nix#L21-L24